torchrecipes/audio/source_separation/datamodule/utils.py

```diff
@@ -0,0 +1,65 @@
+from functools import partial
+from typing import (
+    List,
+    Tuple,
+)
+
+import torch
+from torch import Tensor
+
+
+def get_collate_fn(
+    mode: str = "train",
+    sample_rate: int = 16000,
+    duration: int = 4,
+):
+    if mode == "train":
+        return partial(collate_fn, sample_rate=sample_rate, duration=duration)
+    return partial(collate_fn, sample_rate=sample_rate, duration=-1)
```

Comment:
Instead of `partial`, use of [functor](https://en.wikipedia.org/wiki/Function_object#In_Python) would look cleaner.